sidebars.js

```diff
@@ -749,10 +749,11 @@ const sidebars = {
               label: "Locating Nodes",
               id: "protocol/consensus/locating-nodes",
             },
+            //remove ultralight sync from docs
             {
               type: "doc",
-              label: "Ultralight Sync",
-              id: "protocol/consensus/ultralight-sync",
```

Comment:
please add a redirect from https://docs.celo.org/protocol/consensus/ultralight-sync to the new page